src/pages/AddPage.jsx

```diff
@@ -0,0 +1,105 @@
+import { useRef, useState } from 'react';
+import * as S from './addPage.styles';
+import FormHeader from '../components/FormHeader/FormHeader';
+import FormImageUpload from '../components/FormImageUpload/FormImageUpload';
+import FormInputs from '../components/FormInputs/FormInputs';
+import FormTags from '../components/FormTags/FormTags';
+
+const AddPage = () => {
+  const fileInputRef = useRef(null);
+  const [imagePreview, setImagePreview] = useState(null);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+  const [price, setPrice] = useState('');
+  const [tagInput, setTagInput] = useState('');
+  const [tags, setTags] = useState([]);
+
+  const isFormValid =
+    imagePreview &&
```

Comment:
💊 제안
요구사항에서 이미지는 필수값이 아니라서 해당 조건에서 빼셔도될 것 같아요!
```suggestion
  const isFormValid =
```